source/common/http/conn_manager_impl.cc

```diff
@@ -320,7 +320,12 @@ Network::FilterStatus ConnectionManagerImpl::onData(Buffer::Instance& data, bool
     redispatch = false;
 
     try {
-      codec_->dispatch(data);
+      auto status = codec_->dispatch(data);
```

Comment:
Similar comments here about comments/control flow as I pointed out above in codec_client

Reply:
And done!